_harp/apply/index.ejs

```diff
@@ -2,4 +2,8 @@
 	<div class="container apply-page">
 		<%- partial("_apply.md") %>
 	</div>
+	<div align="center" class="programmes-info">
+		<a href="/apply/london"><button class="button-three">LONDON</button></a>
+		<a href="/apply/nazareth"><button class="button-three">NAZARETH</button></a>
```

Comment:
How about I move this section to the campus-specific pages?
> Before beginning a Founders & Coders programme, you will need to complete our prerequisites. Please apply as soon as you decide you are interested - we do not expect you finish the course prerequisites before putting in your application. But you should make a sincere effort to start this work before we conduct interviews.

Using a partial to avoid duplication.